### PR TITLE
Allow user to start story test with intention to review

### PR DESCRIPTION
**Pull request by andrewyguo**
## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=2a59105407934e1aae7d18fdf93ac775)

## Implementation description

- Add option for user to check wants review when leveling up or adding a new language 
- Notify admin of user intention to be assigned language for review 

## Steps to test

1. Level up a review language, this should create a new story test 
2. add new language, click the I want to be assigned as reviewer option 
3. Go to admin page, confirm that for both of these tests there is an alert indicating that the user would like to be assigned a language for review

## Checklist

- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ x] For backend changes, I have run the appropriate linters: `docker exec -it planet-read_py-backend_1 /bin/bash -c "black . && isort --profile black ."` and I have generated new migrations: `flask db migrate -m "<your message>"`
- [ x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by andrewyguo** [comment]
I still need to add the functionality to display default levels for the user. 

1. Is there an option to set default levels in Chakra UI? So far, I only see the ability to add a placeholder. Not sure how to select a default level. 

2. Also, where should the information for the user's default level come from? Do we need to make a query to the Users table?

File: frontend/src/components/userProfile/ApprovedLanguagesTable.tsx [hunk 1]
```diff
@@ -217,13 +223,18 @@ const ApprovedLanguagesTable = ({
     closeAddNewLanguageModal(false);
   };
 
-  const onUserAddNewLanguage = async (newLanguage: string) => {
+  const onUserAddNewLanguage = async (
+    newLanguage: string,
+    requestReviewer: boolean,
+  ) => {
+    // eslint-disable-next-line
```
Comment (gaoxk) [inline frontend/src/components/userProfile/ApprovedLanguagesTable.tsx:230]: nit: this line doesnt seem necessary

**Review by gaoxk — APPROVED** [review]
feel free to merge after nit

I'm fine with leaving out the default values for now and seeing what the client has to say about the ux while testing

**Comment by gaoxk** [comment]
oh another nit (dont worry about changing any code for this tho) imo the want_review parameter to the gql mutation doesnt need to be required, it could be default `false`

**Pull request merged**